### DISCUSSION TITLE
how to apply displacement on a deformed configration

### DISCUSSION THREAD

**Discussion by eric199405** [idaholab/moose#26071] [Q&A General]
Hi, everyone. From time=0 to time=1s, the boundary A has been deformed to u1 (it's a field). When time>1s, I want to add u2 (constant) on deformed boundary A with u1 field.
If I only add DirichletBC of u2 when time>1s, the previous u1 field will disappear.

**Comment by lindsayad** [comment]
It's not entirely clear to me what you want but you can use the [Controls system](https://mooseframework.inl.gov/syntax/Controls/) to turn different objects on and off

**Reply by GiudGiud** [reply]
You could rewrite the DirichletBC to apply u2 + u1 instead of just u2. 
Just copy the value of u1 into an auxiliary variable using a CopyAux until t=1s (stop copying then)
